lanelet2_io/package.xml

```diff
@@ -14,7 +14,7 @@
   <test_depend>gtest</test_depend>
   <depend>boost</depend>
   <depend>lanelet2_core</depend>
-  <depend>pugixml</depend>
+  <depend>pugixml-dev</depend>
```

Comment:
This still doesn't work because this entry doesn't exist in the main `rosdep` index. Nearly all CI/testing will resolve dependencies for ROS packages using `rosdep` and this package still doesn't resolve, causing build failures.

Reply:
Nevermind. I just saw your PR ros/rosdistro#21509. Thanks!